### PR TITLE
Refine chapters

### PR DESCRIPTION
**Pull request by eteubert**
- add chapters index
- don't include chapters in episode

### CODE REVIEW

File: lib/radiator_web/views/api/chapters_view.ex [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
+      embed: render_many(assigns[:chapters], __MODULE__, "chapter.json", assigns)
+    })
+  end
+
   def render("show.json", assigns) do
     render(__MODULE__, "chapter.json", assigns)
   end
 
-  def render("chapter.json", %{conn: conn, chapter: chapter, audio: audio}) do
+  def render("chapter.json", %{conn: conn, chapters: chapter, audio: audio}) do
```
Comment (monkeydom) [inline lib/radiator_web/views/api/chapters_view.ex]: why the key chapters if you only render one?, shouldn't it still be chapter?

File: lib/radiator_web/views/api/chapters_view.ex [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
   use RadiatorWeb, :view
 
-  alias HAL.{Document, Link}
+  alias HAL.{Document, Link, Embed}
   alias Radiator.AudioMeta.Chapter
 
+  def render("index.json", assigns) do
+    %Document{}
+    |> Document.add_embed(%Embed{
+      resource: "rad:chapter",
```
Comment (monkeydom) [inline lib/radiator_web/views/api/chapters_view.ex]: haven't looked at the result. But wouldn't it be rad:chapters as container? I'm unclear what a top level container should look like here though.

File: lib/radiator_web/views/api/chapters_view.ex (hunk 1 above)
Comment (eteubert) [reply lib/radiator_web/views/api/chapters_view.ex]: Yep, fixed already. The `render_many` put the chapter under the `chapters` key for some reason, so I rewrote it with a manual `Enum.map` and now I can use the `chapter` key again.

File: lib/radiator_web/views/api/chapters_view.ex (hunk 2 above)
Comment (eteubert) [reply lib/radiator_web/views/api/chapters_view.ex]: I think the resource name is always supposed to be singular. The example in http://stateless.co/hal_specification.html includes `ea:order`, which is a list.

**Comment by eteubert** [comment]
Oh, totally forgot about this PR. I cherry-picked 31b3f94 into #194, the rest seems obsolete now.

**Pull request closed without merging**